src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.cs

```diff
@@ -617,9 +629,7 @@ public override IList<TextChange> GetChanges(SyntaxTree oldTree)
         /// </returns>
         /// <remarks>The values are not affected by line mapping directives (<c>#line</c>).</remarks>
         public override FileLinePositionSpan GetLineSpan(TextSpan span, CancellationToken cancellationToken = default)
-        {
-            return new FileLinePositionSpan(this.FilePath, GetLinePosition(span.Start), GetLinePosition(span.End));
-        }
+            => new(FilePath, GetLinePosition(span.Start, cancellationToken), GetLinePosition(span.End, cancellationToken));
```

Comment:
I'm not sure we landed on using target typed `new` in the compiler codebase. @333fred or @jcouv ?

Reply:
IIRC we discourage it. cc @AlekseyTs

Reply:
The current `.editorconfig` says using target typed new is preferred:

![image](https://user-images.githubusercontent.com/41759/120401351-a15d7f00-c2f4-11eb-9dbf-03f66d519903.png)

If that's not what the compiler team prefers pls update your editorconfig.